re/service.go

```diff
@@ -383,43 +271,35 @@ func (re *re) StartScheduler(ctx context.Context) error {
 	}
 }
 
-func (re *re) newTicker(t time.Duration) *time.Ticker {
-	if re.ticker != nil {
-		return re.ticker
-	}
-	return time.NewTicker(t)
-}
-
 func (re *re) shouldRunRule(rule Rule, startTime time.Time) bool {
-	now := time.Now().Add(timeZone).Truncate(time.Minute)
+	now := time.Now().Truncate(time.Minute)
```

Comment:
You can pass only Schedule `func (re *re) shouldRun(s Schedule) bool`, no need for the whole rule or start time here.

Reply:
This has not been addressed.

Reply:
This should be the method attached to the Rule, so we have `rule.ShouldRun(time.Time)`.